cli/src/cmd_net.rs

```diff
@@ -273,7 +273,7 @@ impl AuthenticatedCmd for CmdBgpAnnounce {
     async fn run(&self, client: &Client) -> Result<()> {
```

Comment:
line 250: "address lot" should be "announce set"

Test:

Initial state:
```
$ oxide system networking bgp announcement list --announce-set as65547-announce
[
  {
    "address_lot_block_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "announce_set_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "network": "198.51.100.0/24"
  },
  {
    "address_lot_block_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "announce_set_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "network": "203.0.113.0/24"
  }
```

Invoke command:
```
$ oxide system networking bgp announce \
    --announce-set as65547-announce \
    --address-lot cloud-pool2 \
    --prefix 203.0.153.0/24
```

Resulting state:
```
$ oxide system networking bgp announcement list --announce-set as65547-announce
[
  {
    "address_lot_block_id": "db12d711-1989-4cac-b556-6cbf097fa4a5",
    "announce_set_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "network": "198.51.100.0/24"
  },
  {
    "address_lot_block_id": "db12d711-1989-4cac-b556-6cbf097fa4a5",
    "announce_set_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "network": "203.0.113.0/24"
  },
  {
    "address_lot_block_id": "db12d711-1989-4cac-b556-6cbf097fa4a5",
    "announce_set_id": "387b0ed7-5aa4-4017-9c88-7343c1285605",
    "network": "203.0.153.0/24"
  }
]
```

Reply:
For clarity, here is the help output:
```
$ oxide system networking bgp announce --help
Announce a prefix over BGP.

This command adds the provided prefix to the specified announce set. It is
required that the prefix be available in the given address lot. The add is
performed as a read-modify-write on the specified address lot.

Usage: oxide system networking bgp announce [OPTIONS] --announce-set <ANNOUNCE_SET> --address-lot <ADDRESS_LOT> --prefix <PREFIX>

Options:
      --announce-set <ANNOUNCE_SET>
          The announce set to announce from

      --address-lot <ADDRESS_LOT>
          The address lot to draw from

      --prefix <PREFIX>
          The prefix to announce

      --description <DESCRIPTION>
          [default: ]

  -h, --help
          Print help (see a summary with '-h')

Global Options:
      --profile <PROFILE>
          Configuration profile to use for commands
```